pkg/fmmon/alert.go

```diff
@@ -0,0 +1,36 @@
+package fmmon
```

Comment:
The `pkg` directory is [generally used for packages that are fit for external consumption](https://github.com/golang-standards/project-layout#pkg). I think this package is far too specific, and should be `internal` instead (if not added to the same `package main`).